cmd/cluster-init/update_jobs.go

```diff
@@ -186,6 +186,10 @@ func generateContainer(image, clusterName string, extraArgs []string, extraVolum
 					},
 				},
 			},
+		}...)
+	}
+	if clusterName == string(api.ClusterBuild01) || clusterName == string(api.ClusterBuild02) {
```

Comment:
Any reason why we don't want `ci-slack-api-url` in `build09`? I think we need this to get alerts sent to slack

Reply:
I am using robusta to relay the alerts to Slack